lib/config-generator.js

```diff
@@ -331,11 +331,31 @@ ConfigGeneraror.prototype = {
 
         return new Promise(function(resolve, reject) {
             var result = [];
-
+            var isRequire = false;
             jsstana.traverse(ast, function(node) {
-                var match = jsstana.match('(or (call define ?) (call define ? ?) (call define ? ? ?))', node);
+                var match = jsstana.match('(or (call define ?) (call define ? ?) (call define ? ? ?) (call ?.define ?) (call ?.define ? ?) (call ?.define ? ? ?) (call require ?) (call require ? ?) (call require ? ? ?) (call ?.require ?) (call ?.require ? ?) (call ?.require ? ? ?))', node);
```

Comment:
Umm, here I have a question (I haven't touched this project for a while and I might be wrong, so please bear with me):

The transpiling process generates file like this one below, does't it?
```
define(["exports", "modal@1.0.0/js/address2.es"], function(exports, foo) {
    foo.log("Hello World!");
});
```

This means, it would never generate calls with the namespace in front of `define`, would it?
If so, calls like `call ?.define ?` won't match.

Thanks,